source/reference/operator/aggregation/out.txt

```diff
@@ -68,10 +68,18 @@ Replace Existing Collection
 If the collection specified by the :pipeline:`$out` operation already
 exists, then upon completion of the aggregation, the :pipeline:`$out`
 stage atomically replaces the existing collection with the new
-results collection. The :pipeline:`$out` operation does not change any
-indexes that existed on the previous collection. If the aggregation
-fails, the :pipeline:`$out` operation makes no changes to the pre-existing
-collection.
+results collection using the following process:
+
+#. A temp collection is created.
+#. Indexes are copied from the destination collection to the temp collection.
+#. Documents are inserted into the temp collection.
+#. The temp collection calls `renameCollection
```

Comment:
? the colleciton calls the renameCollection method?

Let's make it more `$out` centric:
... replaces the existing collection with the new results collection.  Specifically, the :pipeline:`$out` operation
#. Creates a temp collection.
#. Copies the indexes from the existing collection to the temp collection.
#. Inserts the documents into the temp collection.
#. Calls :method:`db.collection.renameCollection` with ``dropTarget: true`` to rename the temp collection to the destination collection.

Reply:
That's definitely better, thanks @kay-kim.